cfme/tests/containers/test_projects_dashboard.py

```diff
@@ -0,0 +1,92 @@
+import pytest
+
+from cfme.containers.provider import ContainersProvider
+from cfme.containers.container import Container
+from cfme.containers.image import Image
+from cfme.containers.service import Service
+
```

Comment:
remove empty line, all of these imports are local.

Reply:
remove the empty line.